src/omniscapeTransformer.py

```
@@ -21,6 +24,51 @@
+
+                except psutil.NoSuchProcess:
+                    pass  # Process already terminated
+            else:  # Unix-like systems
+                import signal
+                try:
+                    os.killpg(os.getpgid(julia_process.pid), signal.SIGTERM)
+                except ProcessLookupError:
+                    pass  # Process already terminated
+        except Exception as e:
+            # Best effort cleanup - don't crash if cleanup fails
+            pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)